<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageNewWallet</class>
 <widget class="QWidget" name="PageNewWallet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FCD535;</string>
  </property>
  <widget class="QFrame" name="NewWalletFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>595</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="showHidePushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>152</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235, 235, 235);
	color: black;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/res/ic/res/ic/passwordHidden.png</normaloff>:/res/ic/res/ic/passwordHidden.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="walletNameLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>105</y>
      <width>251</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235, 235, 235);
	color: black;
    padding-left: 4px;
    padding-right: 4px;	
}
QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly|Qt::ImhNoAutoUppercase|Qt::ImhSensitiveData</set>
    </property>
    <property name="placeholderText">
     <string>Type wallet name</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235, 235, 235);
	color: black;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../res.qrc">
      <normaloff>:/res/ic/res/ic/arowLeft.png</normaloff>:/res/ic/res/ic/arowLeft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="walletPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>155</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235, 235, 235);
	color: black;
    padding-left: 4px;
    padding-right: 4px;	
}
QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhPreferLowercase|Qt::ImhSensitiveData</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Type wallet password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="headerLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: #FCD535;
color: #474D57;
}</string>
    </property>
    <property name="text">
     <string>Create new wallet</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="createWalletPushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>255</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235, 235, 235);
	color: black;
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
    </property>
    <property name="text">
     <string>CREATE NEW WALLET</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="walletPasswordLineEdit_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>205</y>
      <width>201</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235, 235, 235);
	color: black;
    padding-left: 4px;
    padding-right: 4px;	
}
QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhPreferLowercase|Qt::ImhSensitiveData</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Retype wallet password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
